plus/src/main/java/org/apache/calcite/chinook/CalciteConnectionProvider.java

```diff
@@ -44,6 +44,7 @@ public Properties provideConnectionInfo() throws IOException {
     Properties info = new Properties();
     info.setProperty("lex", "MYSQL");
     info.setProperty("model", "inline:" + provideSchema());
+    info.setProperty("conformance", "MYSQL_5");
```

Comment:
why is this change needed?

Reply:
The problem I see is that this is affecting all tests that use this connection provider. Do you really want all of these tests to use MYSQL_5 conformance?

Reply:
Considering the lex used by the connection provider is MySQL, I think it should use MYSQL_5 conformance here, which makes the config consistent. What do you think?

Reply:
Perhaps this would work. I guess the fact that the tests passed shows it's not really a problem.